extensions/amp-viewer-integration/0.1/test/test-touch-handler.js

```diff
@@ -114,21 +114,59 @@ describes.fakeWin('TouchHandler', {}, (env) => {
       expect(messages[0].data.data.type).to.equal('touchstart');
     });
 
-    it('should not forward events marked with `shouldViewerCancelPropagation`', () => {
-      const event = fakeTouchEvent('touchmove');
-      event.stopImmediatePropagation = env.sandbox.spy();
+    describe('event propagation', () => {
+      let event, event2;
+
+      beforeEach(() => {
```

Comment:
To be cautious: Let's reset `event` and `event2` in an `afterEach` block

Reply:
I don't think this is an issue. In my experience memory leaks only occur if we're changing/mocking things in the window/env, additionally the beforeEach should always run :)

Reply:
Shouldn't we at least `env.sandbox.restore()`?

Reply:
Oki sgtm